bazel/deps.bzl

```diff
@@ -82,9 +82,41 @@ cmake(
             """,
         )
 
+def cris_deps_leveldb(prefix = "."):
```

Comment:
We have leveldb preinstalled.
leveldb is widely available on almost all distros, and it has been stable for long, so please don't recompile.

Reply:
We preinstall it via Roaster instead of apt. It means if we are not using Docker, we need to either figure out how to use Roaster or build from the source ourselves, but Roaster is a personal project and it may not very stable. Building and installing leveldb manually from source is even worse because you don't know if someone will make any silly mistakes.

Reply:
Emmm...Seems I haven't installed it in docker yet, only in VM.
You may simply throw that into the lib line:
<img width="1114" alt="image" src="https://user-images.githubusercontent.com/5203025/179827156-cfdbfa0a-dc24-4268-acc6-ecdfb938dd82.png">

Reply:
Then we may submit this PR as is and switch to preinstalled levelDB once we updated the docker. Let's not block this one.